docker/policies/apt_get_missing_yes_flag_to_avoid_manual_input.rego

```diff
@@ -0,0 +1,75 @@
+package appshield.dockerfile.DS021
+
+import data.lib.common
+import data.lib.docker
+
+__rego_metadata__ := {
+	"id": "DS021",
+	"title": "APT-GET Missing '-y' To Avoid Manual Input",
+	"version": "v1.0.0",
+	"severity": "HIGH",
+	"type": "Dockerfile Security Check",
+	"description": "Check if apt-get calls use the flag -y to avoid user manual input.",
+	"recommended_actions": "Add -y flag to apt-get",
+	"url": "https://docs.docker.com/engine/reference/builder/#run",
+}
+
+__rego_input__ := {
+	"combine": false,
+	"selector": [{"type": "dockerfile"}],
+}
+
+short_flags := `(-[^ya..zA..Z]*y[^ya..zA..Z]*)`
+
+long_flags := `(--yes)|(--assume-yes)`
+
+combined_flags := sprintf(`(%s|%s)`, [short_flags, long_flags])
+
+deny[res] {
+	args := get_apt_get[_]
+	res := sprintf("-y flag is missed: %s", [args])
+}
+
+get_apt_get[arg] {
+	run = docker.run[_]
+
+	count(run.Value) == 1
+	arg := run.Value[0]
+
+	is_apt_get(arg)
+
+	not includes_assume_yes(arg)
+}
+
+#checking json array
+get_apt_get[arg] {
+	run = docker.run[_]
+
+	count(run.Value) > 1
+
+	common.contains_in_array(run.Value, {"apt-get", "install"})
```

Comment:
I don't think it works as expected. If I understand correctly, it should be true when `apt-get` and `install` both are contained, right? 

But the following will be true when one of `list` elements is contained.

```
contains(array[_], list[_])
```

Reply:
Also, is there any reason for using `contains`? `==` might be better.